src/lib/components/SFooter.svelte

```diff
@@ -0,0 +1,23 @@
+<div class="flex h-auto w-auto justify-between font-inter">
+    <div class="text-left">
+        <p class="text-lg">UP CSI is a service-</p>
+        <p class="text-lg">oriented organization</p>
+        <p class="text-lg">for student learning</p>
```

Comment:
Maybe we could instead rely on `max-width` here rather than manually writing the line breaks ourselves.